content/docs/practices/rules.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Recording rules
+sort_rank: 5
+---
+
+# Recording rules
+
+A consistent naming scheme for [recording rules](../../operating/rules) makes it
+easier to interpret the meaning of a rule at a glance. It also avoids mistakes by 
+making incorrect or meaningless calculations stand out. 
+
+This page documents how to correctly do aggregation and a suggested naming convention.
+
+## Naming and Aggregation
+
+Recording rules should be of the general form `level:metric:operations`.
+`level` represents the aggregation level and labels of the rule output.
+`metric` is the metric name, and should be unchanged other than stripping
+`_total` off counters when using `rate()`. `operations` is a list of operations
+that were applied to the metric, newest operation first. 
+
+Keeping the metric name unchanged makes it easy to know what a metric is and
+easy to find in the codebase. 
+
+To keep the operations clean `_sum` is excluded if there are other operations.
+Associative operations can be merged (for example `min_min` is the same
+as `min`).
+
+If there is no obvious operation to use, use `sum`.  When taking a ratio by
+doing division, seperate the metrics using `_per_` and call the operation
+`ratio`. 
+
+When aggregating up ratios aggregate up the numerator and denominator
+seperately and then divide. Do not take the average of a ratio or average of an
+average as that is not statstically valid.
+
+When aggregating up the `_count` and `_sum` of a Summary and dividing to
+calculate average obseration size, treating it as a ratio would be unwieldy.
+Instead keep the metric name without the `_count` or `_sum` suffix, and replace
+the `rate` in the operation with `mean`. This represents the average
+observation size over that time period.
+
+Always specify a `by` clause with at least the `job` label when aggregating.
+This is to prevent recording rules without a `job` label, which may cause
+conflicts across different jobs.
+
+## Examples
```

Comment:
I tried that initially, it didn't work as well IMHO so I split it as it is currently. This page will need more work over time.